tests/testthat/test-mcmc.R

```diff
@@ -1,3 +1,61 @@
+# JAGS ----
+
+test_that("JAGS model compile works as expected for an example model", {
+  # The example model here is DualEndpointRW.
+  # The steps undertaken in this unit test correspond to the following commands
+  # in (command line) JAGS:
+  # jags
+  # # and then in an interactive mode
+  # model in "model.bug"
+  # data in "data.R"
+  # compile, nchains(1)
+  # We use JAGS through the rjags R package, so that we do not have to invoke
+  # a system command 'jags' with base::system().
+
+  modfile <- "./_jags/model.bug"
```

Comment:
not sure if this will work in general reliably? better use https://testthat.r-lib.org/reference/test_path.html

Reply:
Sure, Tx!, I will check this.